routes/web3relay.js

```diff
@@ -672,6 +672,7 @@ exports.data = async function(req, res){
               console.log("StatsWeb3: check block: " + block.number);
               var blocktime = (latest.timestamp - block.timestamp) / nblock;
               var hashrate = latest.difficulty / blocktime;
+              res.setHeader('Content-Type', 'application/json')
```

Comment:
Please add the header before if condition so it will be added for each response.

Reply:
Changed in the last commit.